src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -51,8 +52,12 @@ class MockWorkflowManagerActor extends Actor  {
 
   def receive = {
     case SubmitWorkflow(sources) =>
-      sender ! MockWorkflowManagerActor.submittedWorkflowId
-
+      val id = MockWorkflowManagerActor.submittedWorkflowId
+      val response = Try(WorkflowDescriptor(id, sources)) match {
```

Comment:
That actually wasn't a fromTry - in the success case it was passing id back and in the failure it was passing the try's failure